hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/performance/ITestS3AOpenCost.java

```diff
@@ -261,10 +263,13 @@ public void testOpenFileLongerLengthReadFully() throws Throwable {
       }
     },
         always(),
-        // two GET calls were made, one for readFully,
-        // the second on the read() past the EOF
-        // the operation has got as far as S3
-        probe(!prefetching(), STREAM_READ_OPENED, 1 + 1));
+        // Analytics stream: 1 open (persistent connection)
+        // S3AInputStream: 2 opens (reopen on EOF)
+            // two GET calls were made, one for readFully,
+            // the second on the read() past the EOF
+            // the operation has got as far as S3
+        probe(!prefetching() && !isAnalyticsStream(), STREAM_READ_OPENED, 2),
```

Comment:
nit: i think 1+1 instead of 2 as the 3rd parameter was intentional. We might want to revert back to 1+1 this line.